src/main/scala/scala/tools/refactoring/sourcegen/SourceUtils.scala

```diff
@@ -6,7 +6,118 @@ package scala.tools.refactoring
 package sourcegen
 import scala.language.postfixOps
 
-object CommentsUtils {
+object SourceUtils {
```

Comment:
In order to resolve the source incompatibility, you could introduce a trait and extend the old object name and the new object name by the trait.